packages/terra-clinical-data-grid/src/subcomponents/Row.jsx

```diff
@@ -87,6 +94,7 @@ class Row extends React.Component {
         data-row
         data-row-id={rowId}
         data-section-id={sectionId}
+        data-allow-column-highlight={allowColumnHighlighting ? true : null}
```

Comment:
I think this inclusion of data-attribute would require consumers to update Jest snapshot. so will this be considered as non-passive change @benbcai ??

Reply:
@supreethmr We often have needs to make changes that would cause consumers to regenerate jest snapshots and wdio screenshots. We consider these types of changes passive because the primary API contract (props) remains unchanged.